### ISSUE TITLE
Windows XP SP3 shared directory doesnt work 

### ISSUE THREAD

**Issue by SamuelPF8** [utmapp/UTM#5255]
I installed Spice tools, when i insalled Spicewebdav it didnt work, and the only version tha worked was Spice webdav 2.2, the Spice proxy service was installed but i start it and it stops 

**Configuration **
* UTM Version: 4.2.5
* OS Version: 16.4
* Device Model: Ipad air 5
* Is it jailbroken (name jailbreak used)? no 
* How did you install UTM? Altstore

![WhatsApp Image 2023-04-28 at 01 11 38](https://user-images.githubusercontent.com/99107644/235079930-6499ea7a-adae-4899-8019-9d10c5c9ef3d.jpeg)

**Comment by Luca1991** [comment]
I have exactly the same problem on macOS (Ventura 13.4.1).

- Spice webdav 2.4 won't install (it is not compatible with XP)
- Spice webdav 2.2 installs just fine, but crashes after starting the service

@SamuelYK  did you find any solution?

I really need shared folders to be productive with UTM.....

**Comment by SamuelYK** [comment]
In Utm no, but I installed the last version git on WinXp, and I upload my documents from cmd  download from GitHub 

@Luca1991 that was the most efficient way i found

**Comment by Luca1991** [comment]
Thanks @SamuelYK for your reply. I decided to create an ISO with all the files I need and to mount it in the VM.

**Comment by SamuelYK** [comment]
You're welcome  @Luca1991 , I thought the same thing but on iPad it's difficult to use ISO

**Comment by thunermay** [comment]
Is there any update to this? I got the same error message regarding spice-webdavd.

**Comment by Added-Glue** [comment]
I wasn't able to get the Shared Folder to connect automatically, but after all the things I tried this easy solution worked:

> All I had to do was to go to "My Network Places" and click on "Add a network place", then "Choose another network location". In the "Internet or network address field", I typed http://localhost:9843/
> Bingo!

[XP: spice-webdavd failed to start: "privileges to start system service"](https://gitlab.gnome.org/GNOME/phodav/-/issues/19#note_1434113])

**Comment by Yokai-Chz** [comment]
> Is there any update to this? I got the same error message regarding spice-webdavd.

@thunermay I couldn’t find a solution with Shared Folder. My solution was to create a private repository on GitHub and download Git on the VM. It worked.

**Comment by Yokai-Chz** [comment]
> I wasn't able to get the Shared Folder to connect automatically in Windows XP, but after all the things I tried this easy solution worked:
@Added-Glue Did you do it on an iPad? I tried it, but it doesn’t work on my iPad. I saw many solutions, but they only work on Macs.

**Comment by Added-Glue** [comment]
@Yokai-Chz Yes, have it working on an iPad. Heard someone say to make sure it’s connected to a folder on your iPad that isn’t in the cloud, but downloaded locally.
